pkg/apis/service/basic.go

```diff
@@ -31,13 +31,12 @@ func (h Handler) Create(req CreateRequest) (CreateResponse, error) {
 	}
 
 	createOpts := pkgservice.Options{
-		TlsCertified: h.tlsCertified,
+		ModelClient: h.modelClient,
```

Comment:
I prefer not putting them in opts given that we explicitly pass modelClient in most of the code base.

Reply:
updated